test/shuffling_openpmd_test.py

```diff
@@ -1,179 +0,0 @@
-        test_parameters.verbosity = 1
-        test_parameters.data.use_lazy_loading = True
-
-        # Train without shuffling.
-        data_handler = mala.DataHandler(test_parameters)
-        from mala.datahandling.data_repo import data_repo_path
-        original_data_path = os.path.join(data_repo_path, "Be2")
-        data_handler.add_snapshot("Be_snapshot0.in.h5", original_data_path,
-                                  "Be_snapshot0.out.h5", original_data_path, "tr", snapshot_type='openpmd')
-        data_handler.add_snapshot("Be_snapshot1.in.h5", original_data_path,
-                                  "Be_snapshot1.out.h5", original_data_path, "va", snapshot_type='openpmd')
-        data_handler.prepare_data()
-
-        test_parameters.network.layer_sizes = [data_handler.input_dimension,
-                                               100,
-                                               data_handler.output_dimension]
-        test_network = mala.Network(test_parameters)
-        test_trainer = mala.Trainer(test_parameters, test_network,
-                                    data_handler)
-        test_trainer.train_network()
-        old_loss = test_trainer.final_validation_loss
-
-        # Shuffle.
-        test_parameters.data.shuffling_seed = 1234
-        data_shuffler = mala.DataShuffler(test_parameters)
-
-        # Add a snapshot we want to use in to the list.
-        data_shuffler.add_snapshot("Be_snapshot0.in.h5", data_path,
-                                   "Be_snapshot0.out.h5", data_path,
-                                   snapshot_type="openpmd")
-        data_shuffler.add_snapshot("Be_snapshot1.in.h5", data_path,
-                                   "Be_snapshot1.out.h5", data_path,
-                                   snapshot_type="openpmd")
-
-        # After shuffling, these snapshots can be loaded as regular snapshots
-        # for lazily loaded training-
-        data_shuffler.shuffle_snapshots("./", save_name="Be_shuffled*.h5")
-        test_parameters.descriptors.descriptors_contain_xyz = True
-
-        # Train with shuffling.
-        data_handler = mala.DataHandler(test_parameters)
-        # Add a snapshot we want to use in to the list.
-        data_handler.add_snapshot("Be_shuffled0.in.h5", ".",
-                                  "Be_shuffled0.out.h5", ".", "tr",
-                                   snapshot_type="openpmd")
-        data_handler.add_snapshot("Be_shuffled1.in.h5", ".",
-                                  "Be_shuffled1.out.h5", ".", "va",
-                                   snapshot_type="openpmd")
-        data_handler.prepare_data()
-
-        test_network = mala.Network(test_parameters)
-        test_trainer = mala.Trainer(test_parameters, test_network,
-                                    data_handler)
-        test_trainer.train_network()
-        new_loss = test_trainer.final_validation_loss
-        assert old_loss > new_loss
-
-TestDataShuffling().test_training()
```

Comment:
Just for being sure: This test is now integrated into the workflow and the normal shuffling test?

Reply:
Yes!